core/src/main/java/org/infinispan/remoting/transport/Transport.java

```diff
@@ -178,4 +181,23 @@
     */
    Set<String> getSitesView();
 
+   <T> CompletionStage<T> invokeCommand(Address target, ReplicableCommand command,
```

Comment:
with this commands, I was expecting the remaining `invokeRemotely*` to be deprecated. What is the plan?